src/main/ipv-stub/ipv-user-identity.ts

```diff
@@ -20,11 +96,9 @@ export const handler: Handler = async (
   }
 };
 
-function get(event: APIGatewayProxyEvent): APIGatewayProxyResult {
+function get(): APIGatewayProxyResult {
   return {
```

Comment:
I think we'll need Content-Type: application/json header for this to be parsed by Orch

https://github.com/govuk-one-login/authentication-api/blob/fecddf3379f8d3b5a2ea7947a2eee7af057943cf/ipv-api/src/main/java/uk/gov/di/authentication/ipv/services/IPVTokenService.java#L119